tasks/utils.js

```diff
@@ -15,7 +15,7 @@ export default (options) => {
   let builder = new jspm.Builder();
 
   return builder.buildSFX(INFILE, OUTFILE, options);
-}
+};
```

Comment:
Function declarations are blocks and not statements, so shouldn't end with semicolon.